webui/react/src/utils/job.ts

```diff
@@ -14,6 +14,9 @@ export const unsupportedQPosSchedulers = new Set<Api.V1SchedulerType>([
 ]);
 
 export const jobTypeIconName = (jobType: JobType): IconName => {
+  if (jobType === JobType.EXTERNAL) {
+    return 'external';
+  }
```

Comment:
**question:** should `external` be added to `CommandType`? Is this for the detached mode job?

Reply:
External jobs are jobs that are not part of DeterminedAI. For example, in an HPC cluster, jobs running on SLURM Workload Manager that are not launched DeterminedAI will be treated as external jobs. So, `external` will not be part of `CommandType` which includes non-experiment DeterminedAI-created job types.